Sources/Plasma/PubUtilLib/plPipeline/GL/plGLMaterialShaderRef.cpp

```diff
@@ -62,7 +62,7 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 //#define USE_NEW_SHADERS 1
 #define HS_DEBUGGING 1
 
-const char* VERTEX_SHADER_STRING = R"(#version 300 es
+const char* VERTEX_SHADER_STRING = R"(#version 330
```

Comment:
Definitely breaks ES. I wasn't sure if there was a way to substitute in using the preprocessor.